api/src/main/java/org/openmrs/module/patientflags/evaluator/SQLFlagEvaluator.java

```diff
@@ -134,7 +134,6 @@ public FlagValidationResult validate(Flag flag) {
 		return new FlagValidationResult(true);
 	}
 
-	@Override

```

Comment:
Did you intentionally remove the above @override?

Reply:
@dkayiwa Yes I did since override is not needed for implementing an interface method